src/manage_nsfs/health.js

```diff
@@ -446,6 +452,47 @@ class NSFSHealth {
         return res;
     }
 
+    /////////////////////////////
+    // LIFECYCLE HEALTH STATUS //
+    /////////////////////////////
+
+    /**
+     * get_lifecycle_health_status returns the lifecycle rules status based on the status of the latest lifecycle wroker run
+     * on the same host
+     * @returns {Promise<object>}
+     */
+    async get_lifecycle_health_status() {
+        const latest_lifecycle_run_status = await this.get_latest_lifecycle_run_status({ silent_if_missing: true });
+        if (!latest_lifecycle_run_status) return {};
+        return {
```

Comment:
what do we have in `latest_lifecycle_run_status` that we only decide to return the `total_stats`, `lifecycle_run_times`, and `errors`?
It will not be flexible if we want to add additional properties.

Reply:
it contains also bucket_statuses, but I don't need to use it on health
please explain what's not flexible in your opinion

Reply:
In case you want to expose more properties in the future.
You can either choose the properties you want to pass or omit the ones that you don't want to pass (`bucket_statuses `).
I was asking this to make sure that it was considered.

Reply:
I chose the properties I wanted to reflect, do you still see here something invalid?

Reply:
No, it can be resolved.